shell/platform/darwin/common/framework/Source/FlutterDartProjectHelper.mm

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <Foundation/Foundation.h>
+
+// Finds a bundle with the named `bundleID` within `searchURL`.
+//
+// Returns `nil` if the bundle cannot be found or if errors are encountered.
```

Comment:
This is a declaration comment, so should be on the declaration rather than the definition.